src/main/java/com/aquarium/aquarium_backend/Controllers/UserSensorControllers.java

```diff
@@ -25,6 +26,12 @@ public ResponseEntity<List<UserSensors>> getAllUserSensors() {
     return new ResponseEntity<>(userSensors, HttpStatus.OK);
   }
 
+  @GetMapping("aquarium/{aquariumId}")
+  public ResponseEntity<List<UserSensors>> getSensorsByAquariumId(@PathVariable Long aquariumId) {
+    return new ResponseEntity<>(
+        userSensorService.getUserSensorsByAquariumId(aquariumId), HttpStatus.OK);
+  }
+
```

Comment:
Add exception handling when aquarium with this Id does not exist.

Reply:
Done